databricks_cli/cluster_policies/cli.py

```diff
@@ -0,0 +1,164 @@
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST to /api/2.0/policies/clusters/create.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/policies/clusters/create'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_cli(api_client, json_file, json):
+    """
+    Creates a Databricks cluster Policy.
+
+    The specification for the request json can be found at
+    https://docs.databricks.com/dev-tools/api/latest/policies.html#create
+    """
+    json_cli_base(json_file, json, lambda json: ClusterPolicyApi(api_client).create_cluster_policy(json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST to /api/2.0/policies/clusters/edit.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/policies/clusters/edit'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def edit_cli(api_client, json_file, json):
+    """
+    Edits a Databricks cluster Policy.
+
+    The specification for the request json can be found at
+    https://docs.databricks.com/dev-tools/api/latest/policies.html#edit
+    """
+    if not bool(json_file) ^ bool(json):
+        raise RuntimeError('Either --json-file or --json should be provided')
+    if json_file:
+        with open(json_file, 'r') as f:
+            json = f.read()
+    deser_json = json_loads(json)
+    ClusterPolicyApi(api_client).edit_cluster_policy(deser_json)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--policy-id', required=True, type=ClusterPolicyIdClickType(),
+              help=ClusterPolicyIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_cli(api_client, policy_id):
+    """
+    Removes a Databricks cluster policy given its ID.
+
+    Use ``databricks cluster_policiess get --policy-id POLICY_ID`` to check termination states.
+    """
+    ClusterPolicyApi(api_client).delete_cluster_policy(policy_id)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--policy-id', required=True, type=ClusterPolicyIdClickType(),
+              help=ClusterPolicyIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_cli(api_client, policy_id):
+    """
+    Retrieves metadata about a cluster policy.
+    """
+    click.echo(pretty_format(ClusterPolicyApi(api_client).get_cluster_policy(policy_id)))
+
+
+def _cluster_policies_to_table(policies_json):
+    ret = []
+    for c in policies_json.get('policies', []):
+        ret.append((c['policy_id'], truncate_string(c['name']), c['definition']))
+    return ret
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Lists Cluster Policiess.')
```

Comment:
Can we run a quick `s/Policiess/Policies/g` on this file?

Reply:
fixed. Thanks